web-app/js/portal/cart/BodaacDownloadHandler.js

```diff
@@ -77,8 +77,7 @@ Portal.cart.BodaacDownloadHandler = Ext.extend(Portal.cart.DownloadHandler, {
                 filters: collection.getFilters()
             });
 
-            var wmsLayer = collection.getLayerState().getSelectedLayer();
-            return wmsLayer._buildGetFeatureRequestUrl(
+            return OpenLayers.Layer.WMS.prototype._buildGetFeatureRequestUrl(
```

Comment:
If we're calling out to this method it probably shouldn't be named with an underscore any more. It is no longer "private"/"internal" as an underscore indicates.